src/Controls/src/Core/Compatibility/Handlers/FlyoutPage/iOS/PhoneFlyoutPageRenderer.cs

```diff
@@ -87,8 +87,8 @@ public void SetElement(VisualElement element)
 
 			_clickOffView = new UIView();
 			_clickOffView.BackgroundColor = new Color(0, 0, 0, 0).ToPlatform();
-			Presented = ((FlyoutPage)element).IsPresented;
 			_viewHandlerWrapper.SetVirtualView(element, OnElementChanged, false);
+			Presented = ((FlyoutPage)element).IsPresented;
```

Comment:
This fixes an NRE caused by setting `Presented` before the `_viewHandlerWrapper` has been set.